content/en/docs/reference/instrumentation/metrics.md

```diff
@@ -5,10 +5,9 @@ description: >-
   Details of the metric data that Kubernetes components export.
 ---
 
-## Metrics (v1.26)
 
-<!-- (auto-generated 2022 Nov 09) -->
-<!-- (auto-generated v1.26) -->
+## Metrics (auto-generated 2022 Nov 01)
```

Comment:
If I understand correctly, this page was generated for v1.25, right? Are you switching to 1.25 the latest commit generated for this file?  Is this date correct?

Reply:
Hey, 

Yes, we are switching to v1.25 because v1.26 is still not released. The issue for the reference https://github.com/kubernetes/website/issues/37922 . 

<img width="975" alt="Screenshot 2022-11-18 at 1 44 57 PM" src="https://user-images.githubusercontent.com/51878265/202654218-88104024-0c47-4570-b7b9-b02597448bc8.png">

Reply:
I'm a little confused, why this time: `2022 Nov 01`? Is this date automatically generated?

		data := templateData{
			AlphaMetrics:     sortedMetrics["ALPHA"],
			StableMetrics:    sortedMetrics["STABLE"],
			GeneratedDate:    time.Now(),
			GeneratedVersion: fmt.Sprintf("%v.%v", major, parseMinor(minor)),
		}
From the point of view of the k8s code, this date should be the time when the script runs.
https://github.com/kubernetes/kubernetes/blob/3f823c0daa002158b12bfb2d53bcfe433516659d/test/instrumentation/documentation/main.go#L150

Reply:
Great point, I am not really sure about it, I just reverted the PR as mentioned the issue. So what should the fix be?
cc @sftim

Reply:
I seem to know the reason, we have to ask whether the original PR https://github.com/kubernetes/website/pull/37522 was generated based on the v1.25 or v1.26?

/cc @logicalhan  @sftim

Reply:
The generator code, which you can read, reported the date that the files were generated.

Because we're reverting, the date will jump back to the date that _this_ older revision of the document was generated. And that's OK.